projects/plugins/jetpack/modules/theme-tools/random-redirect.php

```diff
@@ -13,6 +13,8 @@
 
 // phpcs:disable WordPress.Security.NonceVerification -- No changes to the site here, it just redirects.
 
+_deprecated_file( __FILE__, 'jetpack-$$next-version$$', '', esc_html__( 'Starting Jetpack 13.6, Jetpack will no longer support the Random Redirect module.', 'jetpack' ) );
```

Comment:
This file is always loaded, so this will generate a deprecation warning on every request until we remove it. What do you think about something like:

1. Add something lower... after the check for the `random` query arg .... that sets an option that a random redirect was fired off.
2. If that option is true, show an admin notice, so we only ping folks who have sites that have used it. Make dismissable.
3. Add the deprecation function call at the same time that we remove the inclusion of the file in module-extras.php
4. Remove the file in a following release.